src/core/adapters/jira.js

```diff
@@ -50,12 +51,15 @@ function extractTicketInfo(response, host) {
   const type = issuetype.name.toLowerCase();
   const url = `https://${host}/browse/${id}`;
 
+  const descriptionMd =
+    description && j2m.to_markdown(description).replace('|smart-link', '');
+
   return {
     type,
     id,
     title,
-    description,
     url,
+    description: descriptionMd,
```

Comment:
It might be nice to rename the `const` to a slightly less cryptic `markdownDescription` or alternatively renaming the `description` from the `response` to `jiraText` or `jiraDescription` and using the plain `description` for the converted markdown version.